### PR TITLE
[REVIEW] Fix logging

### PR DESCRIPTION
**Pull request by rommelDB**

### CODE REVIEW

**Comment by romeluko** [comment]
rerun tests

**Comment by rommelDB** [comment]
rerun tests

**Review by williamBlazing — CHANGES_REQUESTED** [review]
Needs some work and also missing what we do for `_optimize_skip_data_getSlices`

File: pyblazing/pyblazing/apiv2/context.py [hunk 1]
```diff
@@ -1167,14 +1197,20 @@ def __init__(
...
         self.config_options = {}
         for option in config_options:
             self.config_options[option.encode()] = str(
                 config_options[option]
             ).encode()  # make sure all options are encoded strings
 
+        # This flag allows to differentiate the accessibility of the files
+        # by the worker nodes. Set to True if the files are distributed,
+        # for example in the case of log files.
+        self.local_files = False
```
[the remark this answers is not in the document]
Comment (rommelDB) [reply pyblazing/pyblazing/apiv2/context.py]: It makes sense. Moved to BlazingTable. Thanks.

File: pyblazing/pyblazing/apiv2/context.py [hunk 2]
```diff
@@ -1167,14 +1197,20 @@ def __init__(
         self.lock = Lock()
         self.finalizeCaller = ref(cio.finalizeCaller)
         self.nodes = []
-        self.node_log_paths = []
+        self.mapping_files = {}
```
[the remark this answers is not in the document]
Comment (rommelDB) [reply pyblazing/pyblazing/apiv2/context.py]: Moved to BlazingTable. Thanks.

**Pull request merged**